### DISCUSSION TITLE
[Idea] Improve the display of gift message in BO

### DISCUSSION THREAD

**Discussion by Prestaworks** [PrestaShop/PrestaShop#32057] [Ideas]
### The problem

Prestashop offers the customers the option to get the order gift wrapped and to add a message to the gift wrapping.

This message is displayed in not that great way today.
![current gift message](https://user-images.githubusercontent.com/2902272/230110073-d9894bf5-cddd-4350-9ff5-d1a6b021f0b6.png)
it is hidden without styling in the shipping tab.

### The solution

I suggest that maybe it can be moved to below the labels and get some styling?

![betterdisplaymaybe](https://user-images.githubusercontent.com/2902272/230110633-abf8c6c3-683b-4df0-9bdb-efc09873ff30.png)

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
What do you think @ElodieOS? It looks like a nice improvement of the interface

**Comment by ElodieOS** [comment]
Hello @Prestaworks, 

Yes it's a really good idea thank you for your contribution! You're right, isn't really user friendly to have this information like this. 

Your proposition is great, I think it's simple and effective enough to be used. Maybe we can go further and show better that the order requires a gift package. 

We can apply your proposition in the carrier tab and add the tag in the header like a specific order characteristic.

Wdyt @Prestaworks and @MatShir? 

![image](https://user-images.githubusercontent.com/113457393/230329103-a0206a83-6866-47fa-9acd-c414c205318c.png)

**Reply by MatShir** [reply]
Sounds good to me ! :)

**Comment by Prestaworks** [comment]
Ok, I make this into an issue and fix a PR.

**Reply by MatShir** [reply]
We can convert this discussion into an issue :)

**Reply by Prestaworks** [reply]
great! I fix the PR then :)

**Comment by kpodemski** [comment]
Thanks, Daniel, for the suggestion and PR. I'm closing this discussion, if needed, we can continue discussing the issue. 

#32104
